framework/src/variables/MooseLinearVariableFV.C

```diff
@@ -22,6 +22,7 @@
 #include "FVDirichletBCBase.h"
 #include "GreenGaussGradient.h"
 #include "LinearFVBoundaryCondition.h"
+#include "LinearFVAdvectionDiffusionFunctorDirichletBC.h"
```

Comment:
I guess that operators introducing spatial derivatives are fundamentally either advection or diffusion 🤔

Reply:
Yeah, but @GiudGiud raised a point saying: what if somebody wants to implement something else with a spatial derivative? To be honest I am tempted to restore the original names but not in this PR.